spec/vlan_list_spec.rb

```diff
@@ -2,22 +2,20 @@
 require 'chef/knife/vsphere_vlan_list'
 
 describe Chef::Knife::VsphereVlanList do
-
   let(:datacenter) { double('Datacenter') }
   let(:n1) { double('Network', name: 'n1') }
   let(:n2) { double('Network', name: 'n2') }
 
-  it 'enumerates the vlans' do
-    knife = described_class.new
+  subject { described_class.new }
 
-    expect(knife).to receive(:vim_connection)
-    expect(knife).to receive(:datacenter).and_return(datacenter)
+  it 'enumerates the vlans' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.